paclab/settings.py

```diff
@@ -125,4 +125,12 @@
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
-MEDIA_URL = '/media/'
\ No newline at end of file
+MEDIA_URL = '/media/'
+
+EMAIL_BACKEND = 'django.core.mail.backends.smtp.EmailBackend'
+EMAIL_HOST = 'smtp.gmail.com'
```

Comment:
If possible, use `config()` for `EMAIL_HOST` and `EMAIL_PORT` and give default values (what you have currently can be the default).